### PR TITLE
chore: enable announcement bar, add major release notes page to docs

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Enable the announcement bar feature in `mkdocs.yml` again
- Create new page that has the release notes + developer commentary for major releases of Renovate
- Link to the new page in the announcement bar
- Put new major release notes page in the navbar

## Context:

We released a new major version of Renovate `v33`, so we should highlight this on our documentation site.

I can't test the changes with the development server on Gitpod, as the build fails for some reason: 

- #233

Please test my work on your local workstation by running a local website preview with `make && make serve`.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ theme:
 
   # Allow overriding the Material for MkDocs themes with a 'overrides' directory.
   # https://squidfunk.github.io/mkdocs-material/customization/#setup-and-theme-structure
-  # custom_dir: overrides // Commenting out this config option to disable our announcement bar. The overrides/main.html file has the code for the announcement bar.
```
Comment (viceice) [inline mkdocs.yml:46]: we should probably leave the comment for later reusage?

**Review by viceice — COMMENTED** [review]
otherwise LGTM

**Review by HonkingGoose — COMMENTED** [review]
We had the idea of making a short release notes blog file somewhere. Let's use this PR to hash out what the contents of this file should be.

We also need to decide _where_ the blog should live. 😉

**Review by viceice — APPROVED** [review]

**Comment by rarkins** [comment]
I've added some commentary to https://github.com/renovatebot/renovate/releases/tag/33.0.0

**Review by rarkins — APPROVED** [review]

File: mkdocs.yml [hunk 2]
```diff
@@ -42,8 +42,10 @@ theme:
   name: 'material'
```
Comment (HonkingGoose) [inline mkdocs.yml:42]: Before you merge, please check that the site preview works properly. I can't test this myself, as Gitpod has build erors, and there's no working CodeSpace setup yet.

Comment (viceice) [reply mkdocs.yml:42]: should be working again 🎉

**Review by viceice — APPROVED** [review]
🤷‍♂️

**Comment by rarkins** [comment]
image

**Pull request merged**